UPDATING.md

```diff
@@ -35,17 +36,17 @@ Make sure your `cldr-json` directory is otherwise clean (`git status`)
 
 ### Updating the Repository
 
-1. Run `git checkout -b ...` to create a new branch.
+1. Run `git checkout -b ...` to create a new branch.  Typical practice is to create a new branch for a major version, such as `v44`, and only merge it back to `main` upon release.
```

Comment:
These changes to UPDATING.md are very helpful! In the context of this PR, the history is confusing. Changes to UPDATING.md are actually from a different PR, already merged, https://github.com/unicode-org/cldr-json/pull/71